apps/emails-and-messages/src/modules/event-handlers/check-message-event-type.ts

```diff
@@ -0,0 +1,10 @@
+import { MessageEventTypes, messageEventTypes } from "./message-event-types";
+
+/**
+ *  Returns the event type if it is valid, otherwise undefined.
+ */
+export const checkMessageEventType = (eventType?: string): MessageEventTypes | undefined => {
```

Comment:
Naming can be better, I dont know what it does by reading "check" (so you need the comment). Maybe there is a better name?

Can be `parse` but maybe change it to `isEventTypeValid(e: string): e is MessageEventTypes` ?